src/app/modules/corporate-contributor/component/add-company-modal/add-company-modal.component.ts

```diff
@@ -37,8 +36,8 @@ export class AddCompanyModalComponent implements OnInit {
     this.checkboxText1 = 'Create a complete CommunityBridge profile for your company.';
     this.checkboxText2 = ' Leave unchecked if you do not want to create a full profile now.';
     this.form = this.formBuilder.group({
-      companyName: ['', Validators.compose([Validators.required, NameValidator.isValid])],
-      companyWebsite: ['', Validators.compose([Validators.required, UrlValidator.isValid])],
+      companyName: ['', Validators.compose([Validators.required, Validators.pattern('^[a-zA-Z0-9- ]*$'), Validators.maxLength(60)])],
+      companyWebsite: ['', Validators.compose([Validators.required, UrlValidator.isValid, Validators.maxLength(255)])],
```

Comment:
I assume the UrlValidator.isValid does the right thing here and validate the URL - similar to: https://www.regextester.com/94502

Reply:
@dealako Ohh it has an issue when it path start or end with number. I will update regex.